src/components/templates/product-table/overview.tsx

```diff
@@ -0,0 +1,81 @@
+import clsx from "clsx"
+import { Link } from "gatsby"
+import * as React from "react"
+import { getProductStatusVariant } from "../../../utils/product-status-variant"
+import ListIcon from "../../fundamentals/icons/list-icon"
+import TileIcon from "../../fundamentals/icons/tile-icon"
+import StatusIndicator from "../../fundamentals/status-indicator"
+import Actionables from "../../molecules/actionables"
+import useProductActions from "./use-product-actions"
+
+const ProductOverview = ({ products, toggleListView }) => {
+  return (
+    <>
+      <div className="flex justify-end border-t border-b border-grey-20 py-2.5 pr-xlarge">
+        <div className="inter-small-semibold text-grey-50 flex justify-self-end">
+          <span
+            onClick={toggleListView}
+            className={clsx(
+              "hover:bg-grey-5 cursor-pointer rounded p-0.5 text-grey-40"
+            )}
+          >
+            <ListIcon size={20} />
+          </span>
+          <span
+            className={clsx(
+              "hover:bg-grey-5 cursor-pointer rounded p-0.5 text-grey-90"
+            )}
+          >
+            <TileIcon size={20} />
+          </span>
+        </div>
+      </div>
+      <div className="grid grid-cols-6">
+        {products.map((product) => (
+          <ProductTile product={product} />
+        ))}
+      </div>
+    </>
+  )
+}
+
+const ProductTile = ({ product }) => {
+  const { getActions } = useProductActions(product)
+
+  return (
+    <div className="p-base group rounded-rounded hover:bg-grey-5 flex-col">
+      <div className="relative">
+        <div
+          className={clsx("rounded-base inline-block absolute top-2 right-2")}
+        >
+          <Actionables
+            actions={getActions(product)}
+            triggerClass="hidden-actions group-hover:opacity-100 focus-within:opacity-100 opacity-0 bg-grey-0"
+          />
+        </div>
+        <Link to={`${product.id}`}>
+          <img
+            className="min-h-[230px] block object-fill rounded-rounded"
```

Comment:
should we here use `object-cover` instead of `object-fill` to avoid thumbnails getting squashed out of proportion: 
![image](https://user-images.githubusercontent.com/88927411/152117287-4c5e9447-253f-4943-b157-2df6670579fb.png)

Reply:
it's just that some pictures will appear as cropped, but I think `object-cover` looks better than `object-fill`